statistics/update.go

```diff
@@ -352,6 +352,9 @@ func (h *Handle) dumpTableStatColSizeToKV(id int64, delta variable.TableDelta) (
 		}
 		values = append(values, fmt.Sprintf("(%d, 0, %d, 0, %d, %d)", id, histID, deltaColSize, version))
 	}
+	if len(values) == 0 {
```

Comment:
Why change this?

Reply:
Statistics for table partition is not done yet, so it will meet some error during constructing the SQL.
The problem is, as long as any error is met, the `DumpStatsDeltaToKV` process would stop, then  `TestShowStatsHealthy` will fail.

That error is irrelevant to the whole statistics functional, **we should ignore that error and continue**, that's why I change here. 

This change should be reasonable. @lamxTyler

Reply:
Yes, it is reasonable.